pkgs/development/python-modules/feather-format/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pyarrow
+}:
+
+buildPythonPackage rec {
+  pname = "feather-format";
+  version = "0.4.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+
+    hash = "sha256-RfZ+N0XTlNTxYMptY2u/1Pi2jQEZncFkm25IfT6HiQM=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [ pyarrow ];
+
+  pythonImportsCheck = [ "feather" ];
+
+  meta = {
+    description = "Simple wrapper library to the Apache Arrow-based Feather File Format";
+    homepage = "https://github.com/wesm/feather";
+    license = lib.licenses.apsl20;
```

```suggestion
    license = lib.licenses.asl20;
```
`apsl20` is deprecated.
https://github.com/NixOS/nixpkgs/blob/f9d80213984daaf1a375f4199b4f666ce5bd21d3/lib/licenses.nix#L1321-L1326